backend/de.metas.manufacturing/src/main/sql/postgresql/system/30-de.metas.adempiere.libero/5663510_sys_gh13843_PP_Order_SeqNo_Index.sql

```diff
@@ -0,0 +1,10 @@
+DROP INDEX IF EXISTS pp_order_seqno
+;
+
+CREATE INDEX pp_order_seqno
+    ON pp_order (seqno)
```

Comment:
because of new/updated requirements, IMHO this index won't help

try something like
create index pp_order_datepromised_day on pp_order (TRUNC(datepromised, 'DD'));

because
* when we compute the next seqno, we search by datepromised (truncated to day)
* i expect to be not so many PP_Orders / day.

If there would be many per day, i would suggest to add seqno to index too... but I doubt
```
create index pp_order_datepromised_day_and_seq on pp_order (TRUNC(datepromised, 'DD'), seqno);
```

PS: @metas-ts  wdyt?